packages/block-editor/src/hooks/layout.js

```diff
@@ -32,29 +33,45 @@ import { getLayoutType, getLayoutTypes } from '../layouts';
 
 const layoutBlockSupportKey = '__experimentalLayout';
 
-const canBlockSwitchLayout = ( blockTypeOrName ) => {
+const getLayoutBlockSettings = ( blockTypeOrName ) => {
 	const layoutBlockSupportConfig = getBlockSupport(
 		blockTypeOrName,
 		layoutBlockSupportKey
 	);
 
-	return layoutBlockSupportConfig?.allowSwitching;
+	const {
+		allowSwitching: canBlockSwitchLayout,
+		default: defaultBlockLayout,
+	} = layoutBlockSupportConfig || {}; // TODO: check if this is needed based on the value return by `getBlockSupport`.
+
+	return {
+		canBlockSwitchLayout,
+		defaultBlockLayout,
+	};
 };
 
 function LayoutPanel( { setAttributes, attributes, name: blockName } ) {
-	const { layout = {} } = attributes;
-	const defaultLayout = useSetting( 'layout' );
+	const { layout } = attributes;
+	// TODO: check if a theme should provide default values per `layoutType`.
+	// Fow now we use the values from `flow` (content, width).
```

Comment:
Not sure what you mean here?

Reply:
So the provided by the theme default should be only about `flow` layout, no?

Reply:
maybe a theme would want to use a "grid" layout by default or some other layout we didn't build yet but yeah in most cases, it will be a "flow" one I think.